apps/package.json

```diff
@@ -57,11 +57,9 @@
     "grunt-notify": "0.4.1",
     "grunt-sass": "1.0.0",
     "grunt-strip-code": "0.1.2",
-    "handlebars-runtime": "1.0.12",
```

Comment:
Found it. https://github.com/code-dot-org/blockly/commit/aa36e34371eb4e03f8805577346b06e72c0c9b64. Not clear that we use it even there